linkerd/app/outbound/src/lib.rs

```diff
@@ -427,13 +428,19 @@ impl<A: OrigDstAddr> Config<A> {
                 http_server.into_inner(),
                 h2_settings,
                 drain.clone(),
-                disable_protocol_detection_for_ports.clone(),
             );
 
+            let svc = svc::stack(tcp_server)
```

Comment:
can we give this a better name than `svc`? There are a lot of services in  this scope... How about something like `tcp_detect`?